kured-ds.yaml

```diff
@@ -25,6 +25,12 @@ spec:
       tolerations:
         - key: node-role.kubernetes.io/master
           effect: NoSchedule
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+        - effect: NoSchedule
```

Comment:
Doesn't this obviate the need for `node-role.kubernetes.io/master` one at the top?